src/test/suite/oidc.test.ts

```diff
@@ -0,0 +1,431 @@
+import os from 'os';
+import path from 'path';
+import assert from 'assert';
```

Comment:
For new tests can we use `expect` from chai? We'd like to move to expect eventually (assert is used pretty much everywhere right now though).

Reply:
The only reason it used assert here was to keep it consistent with the rest of the tests. Happy to move this test to expect since we will do the move to chai and mocha, soon-ish.

Reply:
Addressed this as well in my recent commit